guides/deploying-to-github-pages.md

```diff
@@ -0,0 +1,129 @@
+# Deploying to GitHub Pages
+
+GitHub Pages can host the static files that Tableau produces. This guide walks through a production-ready setup that you can adopt for your own site.
+
+## 1. Prepare your Tableau project
+
+1. Ensure your project exposes a `build` alias that calls `tableau.build`. The generator already includes this:
```

Comment:
I'll leave it up to you, but (even tho I personally use this method) I wonder if this step is necessary information for this guide.